### ISSUE TITLE
Training resources 

### ISSUE THREAD

**Issue by ustcwhy** [facebookresearch/ImageBind#31]
Thanks for your wonderful work.
I am very excited about your idea. May I ask the computation budget used to train the largest Imagebind model? How many GPU hour do you use?

**Comment by fabawi** [comment]
I created a simple ImageBind finetuning example using LoRA:
https://github.com/fabawi/ImageBind-LoRA

Make sure you clone it recursively to include the example dataset:
git clone --recurse-submodules -j8 git@github.com:fabawi/ImageBind-LoRA.git

Install the requirements following the instructions provided in this repo, and run train.py

This should log your checkpoints, as well as separate LoRA if you'd like to update the original model without saving all the model params. More examples and finer control to be added soon

I will post some benchmarks in the README soon

**Comment by ustcwhy** [comment]
> I created a simple ImageBind finetuning example using LoRA: https://github.com/fabawi/ImageBind-LoRA
> 
> Make sure you clone it recursively to include the example dataset: git clone --recurse-submodules -j8 [git@github.com](mailto:git@github.com):fabawi/ImageBind-LoRA.git
> 
> Install the requirements following the instructions provided in this repo, and run train.py
> 
> This should log your checkpoints, as well as separate LoRA if you'd like to update the original model without saving all the model params. More examples and finer control to be added soon
> 
> I will post some benchmarks in the README soon

Thanks. But I actually would like to know how much budget used to train ImageBind from the scratch.

**Comment by LinB203** [comment]
> > I created a simple ImageBind finetuning example using LoRA: https://github.com/fabawi/ImageBind-LoRA
> > Make sure you clone it recursively to include the example dataset: git clone --recurse-submodules -j8 [git@github.com](mailto:git@github.com):fabawi/ImageBind-LoRA.git
> > Install the requirements following the instructions provided in this repo, and run train.py
> > This should log your checkpoints, as well as separate LoRA if you'd like to update the original model without saving all the model params. More examples and finer control to be added soon
> > I will post some benchmarks in the README soon
> 
> Thanks. But I actually would like to know how much budget used to train ImageBind from the scratch.

Hi, here to recommend our [work](https://github.com/PKU-YuanGroup/LanguageBind), which is _LanguageBind: Extending Video-Language Pretraining to N-modality by Language-based Semantic Alignment_. We provide [online demos](https://github.com/PKU-YuanGroup/LanguageBind#-demo). **We open source all training and validation code.**
For video just only 16 V100s are needed, if you turn on gradient accumulation then 8 V100s are fine. For depth maps and infrared maps, only 8 V100s are needed.
